ui/config.js

```diff
@@ -17,6 +17,7 @@
 
 // Define custom options configurable by admins for UI
 cloudStackOptions = {
+    version: "4.11.3", // Cloudstack Version
```

Comment:
this is definately not going to be the case. There must be some other way to get this in the code already (about box contains it)

Reply:
Added more comments